### ISSUE TITLE
lldp sender / lldp/cdp update

### ISSUE THREAD

**Issue by garci66** [WLAN-Pi/wlanpi#19]
It could be useful that the wlanpi could not just receive lldpd but also send its own packets to be discovered by switches, etc.

there is a package for linux called "lldpd" which is also CDP  (and nortel / foundry / extreme ) compatible and can run constantly, without the need for Tcpdump. It can also advertise the hostname / ip / etc of the wlanpi itself so as to make the device visible in the lldp tables of the switches.

One could just hang a few wlan pis (each with its own mac or maybe hostname) and then find in the switch the listing of all those ports. 

Also, by sending lldp it can help find switches that could have lldp disabled and just forward packets out. It can keep a list of active neighbors in case there is a hub/dumb-switch and you receive multiple lldp advertisements

**Comment by javincraig** [comment]
I agree, lldpd would be fantastic to have installed by default, I created my own image with it. Here are the simple commands if the maintainers want an easy starting point or anyone else hits this page looking for an answer to wlanpi LLDP CDP server, it can be accomplished by doing the following:
`sudo apt-get install lldpd`
`sudo service lldpd stop`
`sudo nano /etc/default/lldpd`
!Add to the next line keeping the commented out lines, this enables CDP
`DAEMON_ARGS="-c"`
!Create the config file and edit it
`sudo touch /etc/lldpd.conf`
`sudo nano /etc/lldpd.conf`
`configure system hostname wlanpi`

With that, you can see the device with the name of wlanpi on Cisco devices and it is persistent through reboot. 
If you want to see the interface you are on using lldpd, you can use the following:
`sudo lldpcli show neighbors detail`

**Comment by garci66** [comment]
lldpd has a very nice CLI with the ability to export the clients a json / text / etc.. so it should be relatively easy to integrate into the GUI / menu system. Plus it can remain running on the background constantly. And you could even add more usb nics (if doing a cable mapping job for example) and map all ports of a desktop cable drop (for example with two jacks) as long as you're able to log the neighborship on the switch.
